lib/games.js

```diff
@@ -1270,6 +1270,14 @@ export const games = {
       old_id: 'fivem'
     }
   },
+  gta5r: {
+    name: 'Grand Theft Auto V - RageMP',
+    release_year: 2016,
+    options: {
+      port: 22005,
+      protocol: 'ragemp'
+    }
+  },
```

Comment:
Please add this to properly make the table generator reference the note:
```
extra: {
  doc_notes: 'gta5r'
}
```

Like as in:
```
  trackmaniaforever: {
    name: 'Trackmania Forever',
    release_year: 2008,
    options: {
      port: 2350,
      port_query: 5000,
      protocol: 'nadeo'
    },
    extra: {
      doc_notes: 'nadeo'
    }
  },
  ```
```